src/csv.cr

```diff
@@ -113,7 +113,7 @@ class CSV
   # rows.next # => ["one", "two"]
   # rows.next # => ["three"]
   # ```
-  def self.each_row(string_or_io : String | IO, separator : Char = DEFAULT_SEPARATOR, quote_char : Char = DEFAULT_QUOTE_CHAR)
+  def self.each_row(string_or_io : String | IO, separator : Char = DEFAULT_SEPARATOR, quote_char : Char = DEFAULT_QUOTE_CHAR) : Iterator(Array(String))
```

Comment:
I manually updated this and the other `Iterator` reference to their abstract forms. Was previously resolving to the private `CSV::Parser::RowIterator` class.